vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -111,6 +111,18 @@ void FlyoutShadowNode::createView()
     if (!m_updating && instance != nullptr)
       OnFlyoutClosed(*instance, m_tag, false);
   });
+
+  // Set XamlRoot on the Flyout to handle XamlIsland/AppWindow scenarios.
+  if (auto flyoutBase6 = m_flyout.try_as<winrt::IFlyoutBase6>())
```

Comment:
>auto [](start = 6, length = 4)

const

Reply:
Is there any guideline about this? I think if (auto abc = xx) is more nature and it's widely used everywhere. and I didn't see any benefit here.